docs/examples/call-create-message-multipart-api.swadl.yaml

```diff
@@ -0,0 +1,20 @@
+id: create-message-request-workflow
+activities:
+  - execute-request:
```

Comment:
I'm not sure having an example for an API call that is covered by an existing activity is good
It could lead user to think that this is how they can create a msg

Reply:
I agree this will lead to confusion. I'm removing them in the next commit.